src/main/java/com/pusher/rest/Pusher.java

```diff
@@ -9,6 +9,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import com.pusher.rest.data.*;
```

Comment:
Avoid `*` imports. And maintain the nice import ordering we had (which puts all `com.pusher` in our own block at the bottom)